authentication/auth.go

```diff
@@ -122,41 +122,11 @@ type Metadata struct {
 	Default   string `json:"default"`
 }
 
-func isSurveyMetadata(key string) bool {
-	switch key {
-	case
-		"case_ref",
-		"case_type",
-		"display_address",
-		"employment_date",
-		"form_type",
-		"period_id",
-		"period_str",
-		"ref_p_end_date",
-		"ref_p_start_date",
-		"ru_name",
-		"ru_ref",
-		"trad_as",
-		"user_id",
-		"qid",
-		"survey_id",
-		"PARTICIPANT_ID",
-		"FIRST_NAME",
-		"BLOOD_TEST_BARCODE",
-		"SWAB_TEST_BARCODE",
-		"TEST_QUESTIONS",
-		"sds_dataset_id",
-		"WINDOW_START_DATE",
-		"WINDOW_CLOSE_DATE",
-		"PORTAL_ID",
-		"PARTICIPANT_WINDOW_ID":
-
-		return true
-	}
-	return false
+func (metadata *Metadata) updateMetadataDefault(newDefault string) {
```

Comment:
I don't think this method is being used at the moment?

Reply:
I created it while implementing a solution to removing duplicate metadata items. I left it in because it could be used in the future. But since it's not used there may be no point and can be recreated easily if necessary, I removed it 👍